src/html/shared/swiper.html

```diff
@@ -0,0 +1,58 @@
+<div class="c-swiper">
+    <div class="o-wrapper">
+        <div class="c-swiper__header">
+            <h1>Why choose Shopiflight?</h1>
+        </div>
+
+        <div class="swiper-container u-hide@tablet">
+            <div class="swiper-wrapper">
+                <div class="swiper-slide">
+                    <div class="c-swiper__content">
+                        <button class="c-icon  c-icon--huge  c-icon--static">{{sprite("feature")}}</button>
+                        <h3>Feature 1</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+                <div class="swiper-slide">
+                    <div class="c-swiper__content">
+                        <button class="c-icon  c-icon--huge  c-icon--static">{{sprite("feature")}}</button>
+                        <h3>Feature 1</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+                <div class="swiper-slide">
+                    <div class="c-swiper__content">
+                        <button class="c-icon  c-icon--huge  c-icon--static">{{sprite("feature")}}</button>
+                        <h3>Feature 1</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+            </div>
+            <!-- Add Pagination -->
+            <div class="swiper-pagination swiper-pagination-bullets">
+                <span class="swiper-pagination-bullet swiper-pagination-bullet-active"></span>
+                <span class="swiper-pagination-bullet"></span>
+                <span class="swiper-pagination-bullet"></span>
+            </div>
+        </div>
+
+        <div class="c-swiper--desktop  u-hide@mobile-only">
```

Comment:
in the ilk of mobile first I'd recommend the classes:

`u-hide  u-show@tablet`

Reply:
I think i used this because u-show will write display:block, where else in my c-swiper--desktop, i needed display:flex. and if i put !important on it, it will overwrite display:none on mobile. I've changed it to use include mq in my css instead of using the utility classes.